internal/assets/token_pool.go

```diff
@@ -154,7 +154,10 @@ func (am *assetManager) ActivateTokenPool(ctx context.Context, pool *core.TokenP
 		return err
 	}
 
-	_, err = am.operations.RunOperation(ctx, opActivatePool(op, pool))
+	_, err = am.operations.RunOperation(ctx, opActivatePool(op, pool),
+		false, // TODO: this operation should be made idempotent, but cannot inherit this from the TX per our normal semantics
+		//              as the transaction is only on the submitting side and this is triggered on all parties.
```

Comment:
Is this saying we insert the operation but not the transaction on the non-submitter members? That seems odd/incorrect.

FWIW activation is an idempotent operation at the connector level (so shouldn't be problematic if we activate multiple times) - not sure if that changes how we want to handle this path.

Reply:
That was my reading of the code - would appreciate if you can confirm for me.

It was based on reading this:
https://github.com/hyperledger/firefly/blob/7067eb57b3b3c74a6109dd2e859e50c016f9b921/internal/definitions/handler_tokenpool.go#L99-L107

Reply:
Looks like we do _eventually_ persist the token pool TX, after the activation completes:
https://github.com/hyperledger/firefly/blob/7067eb57b3b3c74a6109dd2e859e50c016f9b921/internal/events/token_pool_created.go#L149

So in the happy path, you do end up with a TX on all members, and the activation operations end up showing underneath it.

But it feels like we should probably persist the TX earlier, before we start creating operations.

Reply:
https://github.com/hyperledger/firefly/issues/1411